libraries/tests/mbed/serial_nc_rx/main.cpp

```diff
@@ -2,29 +2,36 @@
 #include "test_env.h"
 
 int main() {
+    Serial *pc = new Serial(USBTX, USBRX);
```

Comment:
I've had some issues with flushing the buffer in the past. The problem is we don't currently have a blocking call for "flush the serial buffer". There's still the issue of not knowing when the buffer has been emptied and transmitted.